addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/environment-sc-service.js

```diff
@@ -204,7 +213,15 @@ class EnvironmentScService extends Service {
           rev: existingEnvRecord.rev || 0,
           status: SageMakerStatusMap[sagemakerRealtimeStatus[key]].toUpperCase(),
         };
-        await this.update(requestContext, newEnvironment);
+        try {
+          // Might run into situation where the environment was just updated and rev number does not match
+          // Log the error and skip the update for now
+          // The next invocation of poll and sync will do the sync if it's still needed
+          await this.update(requestContext, newEnvironment);
+        } catch (e) {
+          console.log(`Error updating record ${existingEnvRecord.id}`);
```

Comment:
Do you need to propagate  this exception? if so, then you don't want to log it here and instead use the boom conventions for throwing errors.